README.md

```diff
@@ -151,6 +152,8 @@ Further Information
 | elasticsearch_indices_segments_count                                  | gauge     | 1           | Count of index segments on this node
 | elasticsearch_indices_segments_memory_bytes                           | gauge     | 1           | Current memory size of segments in bytes
 | elasticsearch_indices_settings_stats_read_only_indices                | gauge     | 1           | Count of indices that have read_only_allow_delete=true
+| elasticsearch_indices_settings_total_fields                           | gauge     |             | Index setting value for index.mapping.total_fields.limit
+| elasticsearch_indices_mappings_stats_field count                      | gauge     | 1           | Count of fields
```

Comment:
Is this a duplicate of the one above?

Reply:
No one is the setting for total fields allowed in an index.  The other is the actual number of mapped fields in the index.

This allows an alarm to be setup for a index approaching it's field mapping limit

Reply:
Ok, so this should probably be named something like `elasticsearch_indices_mappings_stats_max_fields`. Then the calculation would be something like `elasticsearch_indices_mappings_stats_fields / elasticsearch_indices_mappings_stats_max_fields`. Does that make sense?

Reply:
hmm the only thought there is `elasticsearch_indices_mappings_stats` makes it feel like it's a stat about the index itself vs the fact that it's a setting on the index which is why I went with `elasticsearch_indices_settings`.  But I don't feel super strongly about either direction

Reply:
The problem is, I'm confused by what the metric name is. There's a space before the word `count`, which isn't part of the metric name. We don't put `_count` at the end of metrics, as everything is a count of something, and this term is reserved for the counter in histograms and summaries. So, I'm just overall confused by what is being "counted" here. The help text doesn't really describe what it is.

Reply:
yah sorry that was a typo from the other changes I pulled in.  I've updated the README to hopefully be more clear

Reply:
So, now this is defining `elasticsearch_indices_mappings_stats_fields` twice with different meanings. :confused: You can only define a metric name once.

Reply:
ack yah just a typo in there I removed the duplicate)  the metric isn't actually defined twice